server_process_windows.go

```diff
@@ -7,20 +7,14 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-
-	"github.com/sirupsen/logrus"
 )
 
 const serverExecutablePath = "acServer.exe"
 
 func kill(process *os.Process) error {
-	err := process.Kill()
-
-	if err != nil {
-		logrus.WithError(err).Errorf("Initial attempt at killing windows process (process.Kill) failed")
-		return exec.Command("taskkill", "/T", "/PID", fmt.Sprintf("%d", process.Pid)).Run()
-	}
-
+	//apparently running Process.Kill() and THEN Taskill is a no-go
+	//using only taskkill /t /f works instead, so it's pointless to use Process.Kill() at all - Alberto
+	exec.Command("TASKKILL", "/T", "/F", "/PID", fmt.Sprintf("%d", process.Pid)).Run()
```

Comment:
`exec.Command().Run()` returns an error, and we definitely want to check that here. you could do that by the following:
```go
err := exec.Command(...).Run()

if err != nil {
    return err
}

return nil
```

or even more simply, 

```go
return exec.Command(...).Run()
```

Also while I'm here, I don't think the above comments are necessary. We will have this PR and the commit history as a reference should we need to go back and figure out why something changed :)